### DISCUSSION TITLE
Nginx Reverse Proxy configuration

### DISCUSSION THREAD

**Discussion by jeffwrusso** [teslamate-org/teslamate#2113] [Q&A]
Has anyone got Teslamate working behind and Nginx reverse proxy?  If so, would you mind sharing the config?  I have Grafana working fine, but the map doesn't load when loading the Teslamate web UI, and the pages constantly refresh.

**Comment by jdhorner** [comment]
Are you loading this on Safari for mobile (an iPhone?) or Safari in general on macOS? That's a known issue with Safari and websockets.

**Reply by jeff-russo** [reply]
I am actually.  I will try from a Windows computer.  Thanks!

**Comment by dprius** [comment]
please refer to my nginx proxy configuration.
check three options on details tab.
![Screenshot_20211124-132244_Chrome](https://user-images.githubusercontent.com/94948003/143174992-1182735a-353a-4351-84cd-dd827c799a4a.jpg)
![Screenshot_20211124-132304_Chrome](https://user-images.githubusercontent.com/94948003/143175017-379768d9-796f-436f-a441-95baf744ae31.jpg)

**Reply by eN0Rm** [reply]
Do you have the .conf file?

**Reply by LukeSkywalker993** [reply]
https://github.com/adriankumpf/teslamate/discussions/2711

**Reply by thermoscookies** [reply]
Thank you for posting this. I was able to get my map to show properly with this but when trying to add authentication with Authentik, the map goes back to not loading.

I was curious if you successfully added authentication through npm?
